index.html

```diff
@@ -34,7 +34,7 @@
 .column { padding: 20px; }
 #wmd-button-bar { display: none; }
 #wmd-input { float: left; box-sizing: border-box; width: 50%; resize: horizontal; font-size: 14px; border-right: 1px solid #ddd; height: 100%; overflow: y-scroll; }
-#wmd-preview { overflow-y: auto; overflow-x: hidden; font-size: 15px; height: 100%; box-sizing: border-box; }
+#wmd-preview { overflow-y: auto; overflow-x: hidden; font-size: 15px; height: 100%; box-sizing: border-box; min-height: 100vh; }

```

Comment:
I don't remember about this part (I did it long ago), what would `min-height: 100vh;` be useful for?

Reply:
By default the `#wmd-preview` only wraps its content, but `min-height: 100vh;` stretches the `#wmd-preview` to the full height of the browser window. That's required because I set a dark background in the dark mode which otherwise may not fill the screen, depending on the amount of content in `#wmd-preview`.